diff -urN UrbanTerror/ioUrbanTerrorClientSource/code/client/cl_curl.h UrbanTerror-patched/ioUrbanTerrorClientSource/code/client/cl_curl.h
--- UrbanTerror/ioUrbanTerrorClientSource/code/client/cl_curl.h	2007-10-09 02:47:28.000000000 +1100
+++ UrbanTerror-patched/ioUrbanTerrorClientSource/code/client/cl_curl.h	2011-10-20 00:41:11.451512558 +1100
@@ -34,7 +34,7 @@
 #elif defined(MACOS_X)
 #define DEFAULT_CURL_LIB "libcurl.dylib"
 #else
-#define DEFAULT_CURL_LIB "libcurl.so.3"
+#define DEFAULT_CURL_LIB "libcurl.so.4"
 #endif
 
 #if USE_LOCAL_HEADERS
diff -urN UrbanTerror/ioUrbanTerrorServerSource/code/client/cl_curl.h UrbanTerror-patched/ioUrbanTerrorServerSource/code/client/cl_curl.h
--- UrbanTerror/ioUrbanTerrorServerSource/code/client/cl_curl.h	2007-11-18 19:57:50.000000000 +1000
+++ UrbanTerror-patched/ioUrbanTerrorServerSource/code/client/cl_curl.h	2011-10-20 00:41:25.461512566 +1100
@@ -34,7 +34,7 @@
 #elif defined(MACOS_X)
 #define DEFAULT_CURL_LIB "libcurl.dylib"
 #else
-#define DEFAULT_CURL_LIB "libcurl.so.3"
+#define DEFAULT_CURL_LIB "libcurl.so.4"
 #endif
 
 #ifdef USE_LOCAL_HEADERS
